### ISSUE TITLE
Feature Request: Hooks for DYNs / run a specific DYN when a specific event in Revit occurs

### ISSUE THREAD

**Issue by paulmst** [johnpierson/Relay#9]
e.g - if a user has Relay installed, and clicks sync (for example), then a specific dynamo script runs in the background
or,
like your 'Prevent In-Place Family Creation' warning message, which appears when a user clicks 'Model In-Place'

**Comment by johnpierson** [comment]
K, I selfishly want this too so I will update when/if I get the time for it. 👀

**Comment by paulmst** [comment]
when a specific event occurs... or PERIODICALLY would be very nice!

**Comment by brencass** [comment]
This could be tough to implement with the required fail safes, though as long as it is within the dynamics of what is allowed in a revit addon then it should be ok to do. 

Though i would add that anything that is done has the highest thought on performance for users over its implementation. As a script that runs every 15 minutes but takes 5 minutes to run will make users very unhappy. 

Great idea/thought as long as it is thought about correctly on a user perspective.

**Comment by paulmst** [comment]
@brencass of course! Hope you didn't think I was planning on wiping out a quarter of everyone's day! I can see event triggers having much more possibilities than periodically for sure!
